soap/soap.d.ts

```diff
@@ -8,13 +8,19 @@
 declare module 'soap' {
     import * as events from 'events';
 
-    class WSSecurity {
+    class Security {
```

Comment:
I understand why you added Security. but as a side effect for sharing some common members you have created an export on the module `soap` that does not exist. so according to this declaration file, i can write this with no compiler warning:

```ts
import * as soap from "soap";

var s = new soap.Security(); // Runtime error.
```

if you want to enforce common properties use interfaces.